README.md

```diff
@@ -26,13 +25,11 @@ The package is available from CRAN and can be installed by running
 To install the latest development version of the package directly from
 GitHub use the following code:
 
-```
-if (!requireNamespace("remotes", quietly = TRUE)) {
-  install.packages("remotes")
-}
+    if (!requireNamespace("remotes", quietly = TRUE)) {
```

Comment:
not sure if the ticks were meant to be removed but the .rmd seems to be unchanged, unclear how this would render

Reply:
I think it renders okay ??  
![image](https://user-images.githubusercontent.com/10111024/222612638-65ce4b48-c44c-42e7-807b-ed1229ff11b8.png)